doc/src/docs/InputOutputReference/01a-InputOutputReference.md

```diff
@@ -649,7 +649,7 @@ The choice made for this field has important implications for modeling accuracy
 
 <table class="table table-striped">
 <tr>
-<th>Demanh Winhow</th>
+<th>Demand Winhow</th>
```

Comment:
@mjwitte Just made that fix.

On Fri, Jul 31, 2015 at 4:46 PM Michael J. Witte <notifications@github.com>
wrote:

> In doc/src/docs/InputOutputReference/01a-InputOutputReference.md
> <https://github.com/NREL/EnergyPlus/pull/5065#discussion_r36019572>:
>
> > @@ -649,7 +649,7 @@ The choice made for this field has important implications for modeling accuracy
> >
> >  <table class="table table-striped">
> >  <tr>
> > -<th>Demanh Winhow</th>
> > +<th>Demand Winhow</th>
>
> @rglazer2 <https://github.com/rglazer2> Hadn't noticed before, "Winhow"
> is also misspelled here --> Window
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NREL/EnergyPlus/pull/5065/files#r36019572>.
>